app/common/src/main/AndroidManifest.xml

```diff
@@ -242,7 +241,7 @@
         <!-- This component is disabled by default (if possible). It will be enabled programmatically if necessary. -->
         <receiver
             android:name="com.fsck.k9.provider.UnreadWidgetProvider"
-            android:icon="@drawable/ic_launcher"
+            android:icon="@drawable/ic_mark_email_unread"
```

Comment:
I can't remember why we explicitly set an icon on this component. I couldn't find an answer in the Git history either. My best guess is that Android versions prior to Android 5.0 used this icon in the system UI to add widgets to the home screen. That seems to no longer be the case. So we can probably just drop this attribute.